templates/types/streaming/fastapi/app/api/routers/models.py

```diff
@@ -112,6 +113,40 @@ def is_last_message_from_user(self) -> bool:
         return self.messages[-1].role == MessageRole.USER
 
 
+class LLamaCloudFileService(BaseModel):
```

Comment:
move to own file?

Reply:
This not look like a service integration rather than data model to me, better to move the code to other place and no need to use the `BaseModel` here